scripts/run.sh

```diff
@@ -38,7 +38,7 @@ done
 ###
 # Users
 ###
-MF_USERS_LOG_LEVEL=info MF_EMAIL_TEMPLATE=../docker/users/emailer/templates/email.tmpl $BUILD_DIR/mainflux-users &
+MF_USERS_LOG_LEVEL=info MF_USERS_ADMIN_EMAIL=admin@mainflux.com MF_USERS_ADMIN_PASSWORD=12345678 MF_EMAIL_TEMPLATE=../docker/users/emailer/templates/email.tmpl $BUILD_DIR/mainflux-users &
```

Comment:
This email template path is non-existent, it should be `../docker/templates/users.tmpl`
I'm resolving this in my PR #1355 but maybe you should fix it here also, I'm not sure what will be meged first.

Reply:
if it is in your PR I'll merge easily if you do it before which is more likely